/**:
  ros__parameters:
    visualize: true

    # Topics for all lidars
    lidar_topics: [lidar_1, lidar_2, lidar_3]

    # path for point clouds in pcd format. Structure: /pcd_directory/<lidar_name>.pcd
    pcd_directory: /../data/demo/

    # logging output
    results_file: demo_results.txt

    # calibration results are stored here (path relative to the dir with multi_lidar_calibrator.py)
    output_dir: /../output/
